packages/coin98-wallet/project.json

```diff
@@ -0,0 +1,48 @@
+{
+  "root": "packages/coin98-wallet",
+  "sourceRoot": "packages/coin98-wallet/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/web:rollup",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/packages/coin98-wallet",
+        "tsConfig": "packages/coin98-wallet/tsconfig.lib.json",
+        "project": "packages/coin98-wallet/package.json",
+        "entryFile": "packages/coin98-wallet/src/index.ts",
+        "buildableProjectDepsInPackageJsonType": "dependencies",
+        "compiler": "babel",
+        "format": ["esm", "umd", "cjs"],
```

Comment:
You will need to remove "umd" from this list for build to work.

Reply:
commented fixed